outermysite/polls/imdbExactApi.py

```diff
@@ -0,0 +1,92 @@
+import requests
+import sys
+import traceback
+from bs4 import BeautifulSoup
+
+def getMovieResults(Id):
+    
+    url = "http://www.imdb.com/title/"+Id+"/?ref_=fn_al_tt_1"
+    
+    try:
+        r = requests.get(url)
+    except requests.exceptions.RequestException as e:
+        print "Unable to connect"
+        raise SystemExit("Slow internet connection")
```

Comment:
Really? When you are running on a server, will `slow internet connection` be an issue?